docs/src/get-started/tutorials/get-started.py

```diff
@@ -0,0 +1,172 @@
+"""
+.. _userdoc-tutorials-get-started:
+
+Get started with rascaline
+==========================
+
+Introduction to the rascaline interface using a molecular crystals dataset.
+
+The dataset
+-----------
+
+The dataset is a small subset dataset used in
+`ShiftML2 dataset <https://pubs.acs.org/doi/pdf/10.1021/acs.jpcc.2c03854>`_.
+containing molecular crystals dataset. There are four crystals-
+one with each of the elements [hydrogen, carbon],
+[hydrogen, carbon, nitrogen, oxygen], [hydrogen, carbon, nitrogen],
+and [hydrogen, carbon, oxygen].
+Each crystal has 10 structure, also denoted by frames, attributed to it.
+Adjacent numbers are adjacent frames. Frame 0 is the geometry-optimised frame.
+The following 9 frames contain atoms that are slightly displaced from
+the geometry-optimised frame. You can obtain the dataset from our
+:download:`website <../../../static/datasets.xyz>`.
+
+We first import the crucial packages and afterewards load the dataset
+using chemfiles.
+"""
+
+import chemfiles
+import numpy as np
+
+from rascaline import SphericalExpansion
+
+
+with chemfiles.Trajectory("../../../static/datasets.xyz") as trajectory:
+    frames = [f for f in trajectory]
+
+print(f"The dataset contains {len(frames)} frames.")
+
+# %%
+# We will not explain here how to use chemfiles in detail, as we only use a
+# few functions here. Briefly, :class:`chemfiles.Trajectory` loads structure
+# data in a convenient format into our example. If you want to learn more about
+# the possibilities take a look at the
+# `chemfiles documentation <https://chemfiles.org>`_.
+# Let us now take a look at the first frame of the dataset.
+
+frame0 = frames[0]
+
+print(frame0)
+
+# %%
+# With ``frame0.atoms``` we get a list of the atoms that make up the zero frame.
+# The `name` attribute returns to us the type of the specified atom.
+
+elements, counts = np.unique([atom.name for atom in frame0.atoms], return_counts=True)
+
+print(
+    f"The zero frame contains "
+    f"{counts[0]} {elements[0]}-atoms, "
+    f"{counts[1]} {elements[1]}-atoms, "
+    f"{counts[2]} {elements[2]}-atoms and "
+    f"{counts[3]} {elements[3]}-atoms."
+)
+
+# %%
+# Calculate a descriptor
+# ----------------------
+#
+# We will now calculate the atomic descriptor for this structure using the
+# SOAP spherical expansion as introduced by
+# `Bartók, Kondor and Csányi <http://dx.doi.org/10.1103/PhysRevB.87.184115>`_
+# For doing so we have to define a set parameters telling the rascaline how "detailed" the
+# spherical expansion should be calculated. These parameters are also called hyper parameters because
+# they are not being optimized in a usual machine learning procedure.
```

Comment:
not sure about this: one can use bayesian optimization or other technics to optimize the hypers. In general I tend to see hyper-parameters as the parameters of the representation, in opposition to parameters of the models. This breaks when dealing with NN, which mix the two.

Reply:
It seems to me that it would be best to define these parameters as "model parameters to be configured before the learning process". The most general and honest definition.

Reply:
I would be fine with calling them model parameters but the slang word unfortunately is hyper parameters.

Also we can remove the sentence about the optimization. This does not really help a new user here.